addons/website_sale/data/product_snippet_template_data.xml

```diff
@@ -23,8 +23,12 @@
                         </t>
                     </div>
                     <div class="o_carousel_product_card_footer card-footer d-flex align-items-center">
-                        <div class="d-block font-weight-bold" t-esc="data['price']"
-                            t-options="{'widget': 'monetary', 'display_currency': website.currency_id}"/>
+                        <div>
+                            <span class="font-weight-bold" t-esc="data['price']"
```

Comment:
Can't this be a t-call instead of 3 copy paste ?

Reply:
Does it worth making a t-call only for the `del` tag? If we want to include the `span` displaying `data['price']` we must handle the fact that in 2 cases we have a `class` attribute set to `font-weight-bold` and in 1 case we don't have such `class` attribute.

Reply:
If only for `del` tag not that much
I just had a look, why not using `font-weight-bold` for all 3 actually?
It doesn't seems more or less beautiful with or without it for that third case imo..
Would be problem solved and avoid those duplicated lines

Reply:
ok ;)